### DISCUSSION TITLE
NEXTSPACE

### DISCUSSION THREAD

**Discussion by theokeist** [ravynsoft/ravynos#439] [Q&A]
Soo there is this attempt to make NeXT like DE called NEXTSPACE, but is only for Linux.
https://github.com/trunkmaster/nextspace , Is ravynOS somehow influenced by this project? Ive read that they included some patches for already existing applications, making DE more functional,

**Comment by mszoek** [comment]
Not directly influenced, no. I'm aware of similar projects like GNUstep, NEXTSPACE, Asahi Linux, NextBSD, PureDarwin, Cocotron, etc. All of them have some useful bits or info (esp Cocotron and NextBSD which I incorporated and extended), but the main influence is MacOS itself.
